core/templates/dev/head/pages/community-dashboard-page/services/translate-text-backend-api.service.ts

```diff
@@ -0,0 +1,49 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview A service to make http calls present inside servicees related
+ * to community-dashboard-page services
```

Comment:
Correct this

Reply:
Oops, didn't modify the copied comment. My bad, I'll fix this

Reply:
@nishantwrp fixed this.